changelog.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Replace esdoc by jsdoc
 - Make configuration readable by user in the configuration file.
+- Component positions are now saved after the layout is automatically generated
```

Comment:
changelogs are use to describe the action/evolution you made.

-> Save component's positions after the layout is automatically generated.

Reply:
done